api/src/main/java/org/datacleaner/api/UsageMeteringMessage.java

```diff
@@ -20,26 +20,44 @@
 package org.datacleaner.api;
 
 import java.io.Serializable;
-import java.util.Collections;
-import java.util.Map;
 
 /**
  * Message that can be used to report a component usage (by the component). Consumer of this message type can
  * use the information provided to gather usage statistics/metering etc.
  */
-public class UsageMeteringMessage implements Serializable {
+public class UsageMeteringMessage implements Serializable, ComponentMessage {
+
+    private static final long serialVersionUID = 42L;
+
+    public static final char DETAILS_SEPARATOR_CHAR = ',';
+    public static final char DETAILS_QUOTE_CHAR = '"';
+
+    private static final String DETAILS_QUOTE_STR = String.valueOf(DETAILS_QUOTE_CHAR);
+    private static final String DETAILS_TWO_QUOTES_STR = DETAILS_QUOTE_STR + DETAILS_QUOTE_STR;
```

Comment:
They put it well - "there is no formal specification in existence, which allows for a wide variety of interpretations of CSV files." - which basically means it's just a matter of following _some_ standard. Could we for the moment settle on the MetaModel standard? Maybe you can refer to MetaModel's ```CsvConfiguration.DEFAULT_ESCAPE_CHAR``` (which we might change in the future, although it would be a pretty "surprisey" thing to do for people who rely on that constant).